admin/controllers/service-manager.js

```diff
@@ -139,15 +139,15 @@ const getCheckWindows = async (req, res, next) => {
 
   // Get current check windows
   try {
-    checkWindows = await checkWindowDataService.fetchCurrentCheckWindows(sortField, sortDirection)
-    checkWindowsCurrent = checkWindowService.formatCheckWindowDocuments(checkWindows, true)
+    checkWindows = await checkWindowDataService.sqlFetchCurrentCheckWindows(sortField, sortDirection)
```

Comment:
I thought we agreed to use 'find' methods in data services?

If so, this might be called `sqlFindCurrent()` or perhaps `sqlFindOneCurrent` or something similar - I am unsure how there could be more than one check window open at any one time.   It sounds like a bug.

Reply:
yep.  this is what I pointed out the other day - this data service follows no convention at all.  nor do the controller methods etc.  its been an absolute pig to refactor.  Hence why i just prefixed the existing methods with `sql`